manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala

```diff
@@ -71,75 +90,42 @@ class TransferController(
     *
     * Batch requests are ordered by their IDs for now.
     */
-  private def lookupSummaries(
```

Comment:
This was mostly broken. Rewrote it so it wouldn't try to shove all the logic into a single DB transaction.